cmd/controller/app/server.go

```diff
@@ -278,6 +289,18 @@ func Run(o *options.ControllerRunOptions, ctx context.Context) error {
 	go func() {
 		<-mgr.Elected()
 
+		kubeInfomers.Start(ctx.Done())
+		clusterInformers.Start(ctx.Done())
+
+		if ok := kubecache.WaitForCacheSync(ctx.Done(),
```

Comment:
this wait does not seems to be necessary

Reply:
I move the waitforcachesync func outside the sync controller. As I want to just use the lister interface in sync controller.